easybuild/easyconfigs/v/virtualenv/virtualenv-20.23.1-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'PythonBundle'
+
+name = 'virtualenv'
+version = '20.23.1'
+
+homepage = 'https://python.org/'
+description = "A tool for creating isolated virtual python environments."
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+toolchainopts = {'pic': True}
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('hatchling', '1.18.0'),
+]
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('Python-bundle-PyPI', '2023.06'),
```

Comment:
This dependency seems unnecessary.`virtualenv` only requires `distlib`, `filelock` and `platformdirs`.

Reply:
Update `virtualenv` to only depend on `distlib`, `filelock` and `platformdirs`  in https://github.com/easybuilders/easybuild-easyconfigs/pull/18208/commits/a160ed6f88d57b1e4898d5fb436e311096f85091